pyiron_atomistics/testing/randomatomistic.py

```diff
@@ -9,6 +9,11 @@
 from pyiron_atomistics.atomistics.job.interactive import GenericInteractive
 from pyiron_atomistics.testing.executable import ExampleExecutable
 from collections import defaultdict
+import pint
+
+
+unit = pint.UnitRegistry()
```

Comment:
Can this be part of `__init__()` otherwise it is always loaded when the module is imported which I guess is not necessary here.

Reply:
like `self.unit`?

Reply:
Yes, or `self._unit` as the user does not need it.